internal/tekton-bundle/tekton-bundle.go

```diff
@@ -58,6 +77,22 @@ func getPipelineBundlePath(isOpenshift bool) string {
 	return tektonPipelinesKubernetesBundleDir
 }
 
+func getTasksBundlePath(isOpenshift bool) string {
+	if isOpenshift {
+		return filepath.Join(tektonTasksOKDBundleDir, "kubevirt-tekton-tasks-okd-"+common.TektonTasksVersion+".yaml")
+	}
+	return filepath.Join(tektonTasksKubernetesBundleDir, "kubevirt-tekton-tasks-kubernetes-"+common.TektonTasksVersion+".yaml")
+}
+
+func readFile(fileName string) ([][]byte, error) {
```

Comment:
`code style` - Can you remove this function and just use `os.ReadFile()` instead?
You can change the signature of `decodeObjectsFromFiles()` to take variadic arguments `...[]byte`

Reply:
Thanks! done.